tajo-core/src/main/java/org/apache/tajo/parser/sql/SQLAnalyzer.java

```diff
@@ -1325,6 +1325,9 @@ public Expr visitCreate_table_statement(Create_table_statementContext ctx) {
 
       if (checkIfExist(ctx.USING())) {
         String fileType = ctx.storage_type.getText();
+        if (fileType.equals("csv")) {
+          throw new TajoRuntimeException(new SQLSyntaxError("Creating csv storage type table is disallowed from 0.12."));
```

Comment:
It would be better to let users know that they use ```text``` instead of ```csv```. In addition, I think it would be better if a specific version name is not included in the error message for maintenance issue.